docker-compose.yml

```diff
@@ -16,4 +16,4 @@ services:
   knight-todo:
     build: .
     ports:
-      - "127.0.0.1:6080:6031"
+      - "127.0.0.1:6080:6031"
```

Comment:
this file shouldn't be present in this PR at all. You should have the necessary minimum amount of files in the PR for the task.